### ISSUE TITLE
Bad URI Lines

### ISSUE THREAD

**Issue by iDevelopper** [ikhvorost/M3U8Decoder#2]
Hello,

I found that bad uri are returned from this m3u8 file (BBC Radio 2):
```m3u8
#EXTM3U
#EXT-X-VERSION:3
## Created with Unified Streaming Platform(version=1.8.4)
#EXT-X-MEDIA-SEQUENCE:266994119
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-TARGETDURATION:6
#USP-X-TIMESTAMP-MAP:MPEGTS=3013867424,LOCAL=2024-02-24T08:12:35.200000Z
#EXT-X-PROGRAM-DATE-TIME:2024-02-24T08:12:35.200000Z
#EXTINF:6.4, no desc
bbc_radio_fourfm-audio=96000-266994119.ts
#EXTINF:6.4, no desc
bbc_radio_fourfm-audio=96000-266994120.ts
#EXTINF:6.4, no desc
bbc_radio_fourfm-audio=96000-266994121.ts
#EXTINF:6.4, no desc
bbc_radio_fourfm-audio=96000-266994122.ts
#EXTINF:6.4, no desc
bbc_radio_fourfm-audio=96000-266994123.ts
```

After decoding, the playlist dict contains:
```
  ▿ uri : 7 elements
    - 0 : "## Created with Unified Streaming Platform(version=1.8.4)"
    - 1 : "#USP-X-TIMESTAMP-MAP:MPEGTS=3022507424,LOCAL=2024-02-24T08:14:11.200000Z"
    - 2 : "bbc_radio_fourfm-audio=96000-266994134.ts"
    - 3 : "bbc_radio_fourfm-audio=96000-266994135.ts"
    - 4 : "bbc_radio_fourfm-audio=96000-266994136.ts"
    - 5 : "bbc_radio_fourfm-audio=96000-266994137.ts"
    - 6 : "bbc_radio_fourfm-audio=96000-266994138.ts"```

**Comment by ikhvorost** [comment]
Hi! Thanks, I will look at.

**Comment by ikhvorost** [comment]
Fixed in [1.2.0](https://github.com/ikhvorost/M3U8Decoder/releases/tag/1.2.0). Thanks.
